Tests/conf.json

```diff
@@ -5244,7 +5244,8 @@
         "Recorded Future Test": "Issue 26741",
         "Git_Integration-Test": "Issue 37800",
         "RecordedFutureFeed - Test": "Issue 43923",
-        "RedLockTest": "Issue 24600"
+        "RedLockTest": "Issue 24600",
+        "SymantecEndpointProtection_Test": "Unable to connect to instance"
```

Comment:
Please reopen the [issue](https://github.com/demisto/etc/issues/30157) of the Test-Failure and the reason why he failed